parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -294,8 +294,7 @@ try
     Get-ProvisioningScripts
     Get-LogCollectionScripts
 
-    # TODO(cameissner): uncomment once next CSE package is rolled out to all regions
-    # Disable-KubeletServingCertificateRotationForTags
+    Disable-KubeletServingCertificateRotationForTags

```

Comment:
Could we assure that it is always called before `Write-KubeClusterConfig`?

Reply:
added comment to point this out